samples/broken-links-ok/index.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const functions = require('@google-cloud/functions-framework');
+const GcmSynthetics = require('@google-cloud/synthetics-sdk-broken-links');
+
+/*
+ * This is the server template that is required to run a synthetic monitor in
+ * Google Cloud Functions.
+ */
+
+
+const options = {
+  origin_uri: "https://example.com",
+  // link_limit: 50,
+  // query_selector_all: "a", // https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll
+  // get_attributes: ['href'], // https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute
+  // link_order: "FIRST_N", // "FIRST_N" or "RANDOM"
+  // link_timeout_millis: 30000, // timeout per link
```

Comment:
Is this meant to be commented out?

Reply:
yup! its as an example where you can just uncomment out the code